### DISCUSSION TITLE
Clear Conversations

### DISCUSSION THREAD

**Discussion by PlutoniaX** [acheong08/ChatGPT#478] [Q&A] [answered]
Hi @acheong08 is there a function that can "Clear conversations" - like the button on the chatGPT frontend (bottom left)? I find that after the New Chats bank up, chatGPT stops working until i clear / delete all the chats.

**Comment by acheong08** [comment, marked as answer]
There is a `delete_conversation(id)` function but not one for clearing all of them at the moment. I'll add that feature today since you brought it up

Edit: https://github.com/acheong08/ChatGPT/releases/tag/0.3.0 added the feature

**Reply by PlutoniaX** [reply]
thank you so much this is working really well! =D
